src/subcommands/deploy.ts

```diff
@@ -28,15 +29,18 @@ USAGE:
     deployctl deploy [OPTIONS] <ENTRYPOINT>
 
 OPTIONS:
-        --exclude=<PATTERNS>  Exclude files that match this pattern
-        --include=<PATTERNS>  Only upload files that match this pattern
-        --import-map=<FILE>   Use import map file
-    -h, --help                Prints help information
-        --no-static           Don't include the files in the CWD as static files
-        --prod                Create a production deployment (default is preview deployment)
-    -p, --project=NAME        The project to deploy to
-        --token=TOKEN         The API token to use (defaults to DENO_DEPLOY_TOKEN env var)
-        --dry-run             Dry run the deployment process.
+        --exclude=<PATTERNS>    Exclude files that match this pattern
+        --include=<PATTERNS>    Only upload files that match this pattern
+        --import-map=<FILE>     Use import map file
+    -h, --help                  Prints help information
+        --no-static             Don't include the files in the CWD as static files
+        --prod                  Create a production deployment (default is preview deployment)
+        --project=<NAME|ID>     The project to deploy to
+        --entrypoint=<PATH|URL> The file that Deno Deploy will run
```

Comment:
Why is this a flag now? It's already a positional argument

Reply:
> particularly in the use case where the project has an entrypoint and a project configured, and you want to deploy using another pair of project/entrypoint

That can also be achieved in the combination of flag for project + positional argument for entrypoint right? I understand that it's intuitive, but having fewer flags where possible is also good.

Reply:
Honestly, I think having the entrypoint as positional argument has always been a mistake. Project was also a required parameter yet defined as a flag, so it's always been confusing that one required parameter was added as a flag, and another as a positional argument.

> but having fewer flags where possible is also good.

I agree that in general having fewer _parameters_ is good for simplicity. But this is not the case, as the number of parameters is the same, with the additional confusion that one of them is not named and does not follow the pattern of the rest of the parameters. 

However, I agree that having 2 ways to do the same thing is also confusing. My preferred solution would be to remove the positional argument from the docs and keep its logic only for backwards compat. What do you think?